hugo/layouts/partials/footer.html

```diff
@@ -7,8 +7,8 @@
       <a class="-icon-twitter" href="https://twitter.com/ShineText" target="_blank"></a>
     </div>
     <div class="page-links">
-      <a href="https://premium.shinetext.com">Shine Premium</a>
-      <a href="https://images.ctfassets.net/awpxl2koull4/YaA04LXbkyUsyGggOS22I/f7734cca463599e1e359e16c3d7c21f2/Shine2019HustleAndHeartReport.pdf">Why Self-Care Matters</a>
+      <a href="https://join.shinetext.com">Join Shine</a>
+      <a href="https://premium.shinetext.com/quiz/start?utm_source=Shine&utm_medium=Blog&utm_campaign=Footer">Self-Care Style Quiz</a>
```

Comment:
I don't know if there are tracking implications to this, but you could also use the `join.shinetext` subdomain for this link too.

Reply:
I think I've been using premium.shinetext.com for all of the links to the quiz across all the sites just to keep all the data on one landing page!